spec/features/clearance/user_signs_in_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+feature "User signs in" do
+  scenario "with valid email and password" do
+    create_user "user@example.com", "user", "password"
+    sign_in_with "user@example.com", "password"
+
+    expect_user_to_be_signed_in
+  end
+end
+
+def create_user(email, username, password)
```

Comment:
I personally keep factory methods inside of the top level describr/feature block. But since this is generated code, it might be unnecessary.